gef.py

```diff
@@ -5545,8 +5552,8 @@ def __create_new_structure_template(self, structname: str, fullname: str) -> Non
             "    _values_ = []",
             ""
         ]
-        with open(fullname, "w") as f:
-            f.write(os.sep.join(template))
+        with fpath.open("w") as f:
+            f.write(os.linesep.join(template))
```

Comment:
Because of changing to use `os.linesep`, the code on line 5549

```
            "class ", structname, "(Structure):",
```

will end up as 

```
class
structname
(Structure):
```

causing a `SyntaxError` when trying to load the struct definition:

```
[!] Command 'pcustom show' failed to execute properly, reason: invalid syntax (foo_t.py, line 3)
```

I can't suggest the change in this review because these lines weren't part of the commit, but this should work

```py
f"class {structname}(Structure)"
```

Reply:
good find.

We could also just turn the template into one large string e.g.
```python
template = ("from ctypes import *"
"\n"
f"\nclass {structname}(Structure)"
...
```

or of course do something like
```python
template = f"""from ctypes import *
...
class {structname}(Structure)
...```

Reply:
Very cool, that bug was actually there for a while. Another good catch

Reply:
Fixed

Reply:
lgtm